setup.py

```diff
@@ -305,6 +305,7 @@
         'IMS10BULLETIN = obspy.io.iaspei.core',
         'EVT = obspy.io.sh.evt',
         'FOCMEC = obspy.io.focmec.core',
+        'PHA = obspy.io.hypodd.pha'
```

Comment:
Hmm, maybe `HYPODDPHA` instead? Might be more meaningful but also longer, so dunno..

Reply:
I have no preferences here... just a bit of work

Reply:
... no work at all. I did not change the file names just the format name.

Reply:
> I did not change the file names just the format name.

Yep, that's what I meant. :+1: